estate/models/users.py

```diff
@@ -7,5 +7,5 @@ class User(models.Model):
     property_ids = fields.One2many(
         "estate.property",
         "salesperson_id",
-        domain=[('state', 'in', ['new','offer_received'])],
+        domain=[('state', 'in', ['new','offer_received'])], 
```

Comment:
Check your linter on these changes
```suggestion
        domain=[('state', 'in', ['new','offer_received'])],
```